### PR TITLE
Display Minetest header when menu_last_game value isn't available anymore

### LINKED ISSUE #9962: The game header image is not displayed when restarting MT after removing a game
##### Minetest version

```
Minetest-5.3.0-dev-82e413789
```

##### Summary
Just have discovered in that build the MT logo is not displayed above the menu after each startup into 'Start Game' tab. It has appeared only after switching to any other tab:

**After the startup:**
Screenshot_20200529_180734

**And further after switching to 'Join Game' it is appeared:**
Screenshot_20200529_180751

Screenshot_20200529_180810

##### Steps to reproduce
Just start up the game engine directly into 'Start Game' tab.

### PR DESCRIPTION
**Pull request by Zughy**
Fixes #9962

## To do

This PR is Ready for Review.

## How to test
- Install a game
- Close MT having that game selected in the local tab
- delete that game folder
- reopen MT
- "Minetest" header should appear

### CODE REVIEW

File: builtin/mainmenu/tab_local.lua [hunk 1]
```diff
@@ -22,6 +22,9 @@ if enable_gamebar then
 	function current_game()
 		local last_game_id = core.settings:get("menu_last_game")
 		local game = pkgmgr.find_by_gameid(last_game_id)
+		if game == nil then
+			mm_texture.reset()
+		end
```
Comment (sfan5) [inline builtin/mainmenu/tab_local.lua:27]: shouldn't this rather happen in the place `current_game()` is first called?

Comment (Zughy) [reply builtin/mainmenu/tab_local.lua:27]: I guess. I mean, this should cover even if someone deletes a game folder when the client is open, but how much likely is it to happen? Let me know

Comment (Zughy) [reply builtin/mainmenu/tab_local.lua]: Never mind, done

**Review by SmallJoker — APPROVED** [review]
Fixes the described issue.

**Pull request merged**